mk/spksrc.toolkit-flags.mk

```
@@ -0,0 +1,39 @@
+ifeq ($(strip $(TOOLKIT_NAME)),)
+TOOLKIT_NAME = syno-$(TOOLKIT_ARCH)
+endif
+
+ifeq ($(strip $(TOOLKIT_DIST)),)
+TOOLKIT_DIST = ds.evansport-$(TOOLKIT_VERS).dev
```

Hum, I guess you mean
```suggestion
TOOLKIT_DIST = ds.$(TOOLKIT_ARCH)-$(TOOLKIT_VERS).dev
```

---

good catch!  now fixed.